alwaysontop/main.js

```diff
@@ -53,9 +53,9 @@ function onAlwaysOnTopWindow(
         });
         jitsiMeetWindow.webContents.send('jitsi-always-on-top', {
             type: 'event',
-            name: 'new-window',
             data: {
-                id: win.id
+                id: win.id,
+                name: 'new-window'
```

Comment:
What is the advantage of putting name within data?

Reply:
Just to be consistent with Jitsi Meet Transport format.